src/derivatives/atlas.md

```diff
@@ -8,45 +8,43 @@ segmentations, and other knowledge annotations such as landmarks defined with
 respect to individual or template brains that are supported by spaces
 such as surfaces and regular grids (images).
 
-For derivatives with atlases in their provenance corresponding to individual subjects,
-the organization follows the standards for BIDS raw and derivatives.
+The organization follows the standards for BIDS raw and derivatives.
 The following entities MAY be employed to specify template- and atlas-derived results:
 
 -    [`tpl-<label>`](../glossary.md#template-entities) is REQUIRED to specify derivatives defining
-     a [template](../common-principles.md).
+     a [template](../common-principles.md).  MUST NOT be used alongside `atlas-<label>`
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to specify derivatives defining an [atlas](../common-principles.md).  MUST NOT be used alongside `template-<label>`
 -    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
      respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
-     BIDS-Derivatives specifications.
+     BIDS-Derivatives specifications.  It is also REQUIRED when [`tpl-<label>`](../glossary.md#template-entities) is used,
+     and the template is NOT serving as the authoritative definition of a space.
```

Comment:
I think defining where and at what time the averaging takes place is still tricky to define and make solid requirements for. For example, for PET data we usually 1) project the 4D PET to e.g. fsaverage (using anatomical data, i.e. sub-XX_space-fsaverage_pet.nii.gz), 2) smooth the data along the cortical surface, and then 3) perform kinetic modeling to produce binding estimates. We then average these binding estimates across subjects (i.e. sub becomes tpl to create a new template existing in fsaverage space). This process works well because it respects individual cortical boundaries by registering individual surfaces to fsaverage. It would instead be suboptimal to first volumetrically smooth each individual data in subject space, produce binding estimates, register all subjects to e.g. the first subject, average the binding estimates across subjects to produce a new template, and then register that template to e.g. fsaverage. But ultimately, the outcome of these two workflows would be the same i.e. average binding estimates across a set of subjects in fsaverage space. But in the first case, this might be considered an atlas on top of tpl-fsaverage (i.e. tpl-fsaverage_atlas-PET_pet.nii.gz) or a new template since we are averaging across subjects (i.e. tpl-PET_space-fsaverage_pet.nii.gz), whereas for the second case it might be considered a template in fsaverage space (i.e. tpl-PET_space-fsaverage_pet.nii.gz). Now if I define a novel PET parcellation/atlas based on this, I would have tpl-fsaverage_atlas-PET_probseg/dseg.nii.gz for case 1 (i.e. only the suffix datatype changes), and for case 2 it would be tpl-PET_space-fsaverage_atlas-PET_probseg/dseg.nii.gz.

In my humble opinion, I suggest that when averaging is taking place for a new datatype e.g. pet (instead of T1w) and a new set of subjects in either a new or already defined space, this becomes a new template existing in that defined space. A new segmentation/atlas may then subsequently be defined on top of this (with a dseg/probseg extension), and will therefore also belong to a specific template and a specific space.

Reply:
Your example seems to me like pretty clearly belonging to ``tpl-fsaverage`` (and your second paragraph is consistent with the changes you made in https://github.com/bids-standard/bids-specification/pull/1856/commits/891bcf01bd9638e37909302c02c314e5648a3c27, which I was planning to respond in our meeting but then got distracted and did not mention.

HOWEVER, there was a point in the meeting where we touched base on this. What we said is, considering that, even though you are working in fsaverage coordinates you are not the original originator of fsaverage, then we should recommend that a new template identifier (such as PS13) is issued.

In the `dataset_description.json` then we should clearly indicate that the new template is in fact aligned with fsaverage. If that is the case, then `sub-01_hemi-L_space-PS13_PET.gii` becomes absolutely equivalent to `sub-01_hemi-L_space-fsaverage_PET.gii`, in both cases signifying PET data on a surface defined in fsaverage/PS13 space, which is the same.

We could (and I like the idea a lot) enforce that space is present in the filenames if the new template does not generate its own coordinate system (which is what @pwighton was basically pointing us at):

```
tpl-PS13/
    tpl-PS13_space-MNI152Lin_T1w.nii.gz
```
This example tells you that you generated a T1w average (whether in a custom space of the study or you moved everything to MNI152Lin should be found in some provenance metadata and/or a paper) that is not the one distributed with MNI152Lin, but in this case it is aligned with that space.

The generation of new template identifiers can be recommended in that case, but this has effects to things like TemplateFlow more than the specification itself. BIDS can perfectly live without this recommendation, nor the combination of tpl- and space- in this use-case.

Why?
- For BIDS, the specification disallowing generating new template identifiers if they are really defined in a different, existing template space, would work as-is. It tells you how to organize the data and the organization is absolutely valid.
- TemplateFlow and other potential services care about the ownership and access rights of stakeholders. At the moment, the developers of TemplateFlow regulate that, but, if it were a "proper" registry, we should not modify say fsaverage adding PS13 when the authors are not exactly the same or the scope of fsaverage is not to cover PS13.

That said, this is already creating difficulties in understanding, so I believe BIDS should observe these nuances and I think adding these mechanisms is a good thing for the spec. But again, it's more a problem of data ownership/access/stewardship than it is of nomenclature or organization.

Happy to work through examples in the call if necessary.

Reply:
Thanks for this reply, @oesteban, and looking forward to work through some of the examples in our call tomorrow. Researching a bit on how already existing T1w templates were generated (https://www.lead-dbs.org/about-the-mni-spaces/) e.g. MNI152 NLIN 2009, 152 brains were nonlinearly registered into 305 space (i.e. MNI305) and then averaged to create a new template i.e. MNI152 NLIN 2009. So I think this conflicts a bit with the current notion that averaging should take place in a new space defined by the 152 brains and not in an already defined template space, as the MNI152 NLIN 2009 template would otherwise not be a new template, but instead either 1) an atlas tpl-MNI305_atlas-MNI152NLIN2009_T1.nii.gz, or 2) a new template defined in an already existing space tpl-MNI152NLIN2009_space-MNI305_T1w.nii.gz. In the PS13 case, alternatively, this should just be a new template tpl-PS13_pet.nii.gz. But adding the space entity seems necessary to differentiate between different output spaces e.g. fsaverage or MNI305?

Reply:
It so happens that fsaverage IS MNI305 with an adjustment of an offset in the through-plane (Z) direction :D.

I'm aware of what you mean, however, the difference between MNI305 and MNI152NLin2009cAsym (and variants) is that the alignment into MNI305 is linear, and a new anatomical reference and registration method where employed (nonlinear, since 2006).

Because the registration is nonlinear, I would be confident to call MNI152NLin2009cAsym a new template w.r.t. MNI305.

Let's pick fsaverage, MNI305 and PS13.

PS13 *is* in fsaverage so long the spatial transform between PS13's maps/surfaces/etc. and fsaverage is identity. However, between MNI305 and fsaverage there is a translation and therefore the transform is not identity.

Between MNI305 and MNI152NLin2009cAsym, the linear part of the transform is identity, but the nonlinear part of the transform (which is not released by MNI because they probably did not even calculate it, I would have not calculated it) is not identity.

Now, I'm going to give you ammo to fire at my argument. Okay, the transform between MNI152NLin2009aAsym, MNI152NLin2009bAsym, MNI152NLin2009cAsym *is* the identity.

-> Yes, but in this case, I believe we can apply the principle explained above and allow them to be separate. Alternatively, we can change (which is not backward compatible) BIDS' existing designation of some standard spaces so that a, b, and c versions of 2009 become just MNI152NLin2009Asym.

Reply:
Thanks, @oesteban - I think your take on this is really helpful. For the PS13 example (and other PET examples in general), we generate the average feature map using often a volumetric stream and a surface-based stream. For the volumetric stream, we have used CVS registration (https://surfer.nmr.mgh.harvard.edu/fswiki/mri_cvs_register) estimating and applying a nonlinear registration from subject space to a CVSMNI152 template provided by FreeSurfer. After this registration, the data is averaged across subjects. To me, this aligns with https://www.lead-dbs.org/about-the-mni-spaces/ stating 'MNI152 NLIN 6th generation: Here, the 152 brains were nonlinearly registered into 305 space. Again, only a T1 template is available.', and so this volumetric part should be defined as a new template wrt. the CVSMNI152 template/space (i.e. tpl-PS13_pet.nii.gz). However, for the surface-based stream we estimate and apply the registration from subject space to fsaverage, and then average across subjects. So the spatial transform between this new surface-based feature map and fsaverage is identity, and would not be defined as a new template, and so should instead be defined as a continuous atlas with a _pet.nii.gz extension (i.e. tpl-fsaverage_atlas-PS13_pet.nii.gz'). If one would estimate a parcellation on top of this, the filename would change to tpl-fsaverage_atlas-PS13_dseg/probseg.nii.gz). Right?  

I think I would be okay with this, and it would not conflict with prior templateflow work/distribution, but would impact the distribution of templates/atlases moving forward. I guess it might be slightly confusing that the same continuous PET data can be considered both a template and an atlas, and so the alternative would be to use the space entity (as you mentioned above), enforcing that space is present in the filenames if the new template does not generate its own coordinate system. Furthermore, I'd be inclined to still use the space entity in the volumetric stream case above, stating that the PS13 template now exists in CVSMNI152 space, similar to the fact that I think the tpl-MNI152 NLIN 6th generation should theoretically also have had the space-MNI305 part included. Then the PS13 data could be distributed as 

tpl-PS13_space-fsaverage_pet.nii.gz
tpl-PS13_space-CVSMNI152_pet.nii.gz
tpl-PS13_space-MNI305_pet.nii.gz
tpl-PS13_space-CVSMNI152_atlas-PS13_dseg.nii.gz